tsdb/engine/tsm1/float.go

```diff
@@ -130,9 +129,7 @@ type FloatDecoder struct {
 	trailing uint64
 
 	br BitReader
-
-	b       []byte
-	breader bytesReader
+	b  []byte
```

Comment:
The reader now takes a `[]byte` instead of `io.Reader` to avoid function invocation and interface overhead.